specifyweb/frontend/js_src/lib/components/Interactions/PrepDialog.tsx

```diff
@@ -136,7 +136,7 @@ export function PrepDialog({
         {commonText.bulkSelect()}
         <Input.Integer
           aria-label={interactionsText.selectedAmount()}
-          className="w-[unset]"
+          className="!w-[unset]"
```

Comment:
fyi: if we have too many issues with the need for `!`, we could explore out of the tailwind helper libraries - https://github.com/specify/specify7/issues/4328

these libs would deduplicate classes, making sure that the place where the class is set last takes precedence over the earlier usages, thus reducing/removing the need for `!` and the associated bugs...